hardware_integration/include/isobus/hardware_integration/can_hardware_interface.hpp

```diff
@@ -38,33 +38,27 @@ class CANHardwareInterface
 	class CanLibUpdateCallbackInfo
 	{
 	public:
-		/// @brief Constructs a `CanLibUpdateCallbackInfo`, sets default values
-		CanLibUpdateCallbackInfo();
```

Comment:
Is it better to have `CanLibUpdateCallbackInfo() = default;` ?

Reply:
I'm lost here. I can't gasp the difference between no constructor at all and a default one: https://stackoverflow.com/questions/68821529/what-is-the-difference-between-constructor-default-and-the-compiler-generated

Reply:
Hmm, I think in this case since there's no protected or private variables it is the same? This is one situation where maybe Sonar has a preference on it

Reply:
Okay, let's see. At least if Sonar works this Sunday haha....

Reply:
Sonar doesn't seem to bother about not having the default constructor it seems?

Reply:
OK, good enough for me then I suppose